spec/rubocop/cop/rspec/pending_without_reason_spec.rb

```diff
@@ -65,6 +65,32 @@
     end
   end
 
+  context 'when pending by pending step without reason ' \
+          'and not inside example with block' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        FactoryBot.define do
+          factory :task do
+            pending { true }
+          end
+        end
+      RUBY
+    end
+  end
+
+  context 'when skipped by skip step without reason ' \
+          'and not inside example with block' do
+    it 'registers no offense' do
+      expect_no_offenses(<<~RUBY)
+        FactoryBot.define do
+          factory :task do
+            skip { true }
+          end
+        end
+      RUBY
+    end
+  end
+
```

Comment:
Thank you so much.

We may want to start by organizing your specs with reference to this section.
In any case, it would be better to start after this PR :+1:

https://relishapp.com/rspec/rspec-core/v/3-12/docs/pending-and-skipped-examples